Counting/script.js

```diff
@@ -0,0 +1,51 @@
+for (let i = 1; i <= 10; i++) {
+    console.log(i)
+;}
+
+for (let i = 10; i >= 1; i--) {
+    console.log(i);
+}
+
+let i = 1
```

Comment:
It's best to reserve 'i' for for loops. I usually see this kind of variable named 'count'.